src/robot/errors.py

```diff
@@ -129,7 +130,10 @@ def get_errors(self):
 
     @property
     def status(self):
-        return 'FAIL'
+        if isinstance(self, ExecutionSkipped):
```

Comment:
Instead of this isinstance check, you could re-implement the `status` property on the `ExecutionSkipped` class.